.github/workflows/preview.yml

```diff
@@ -1,7 +1,15 @@
+# This file generated from a template file maintained in the ivoatex repository.
+# To create and install it into a project repository, do:
+#    make github-preview
+#    git commit
+#    git push
+#
 name: Update PDF Preview
 
 env:
-  doc_name: DALI
+  doc_name   :
```

Comment:
`doc_name` isn't set here, so later in L38 the `${{ env.doc_name }}-draft.pdf` substitution becomes `-draft.pdf`. It "passed" (i.e. didn't run) in the PR because this workflow will only run on pushes to the `main` branch.

Is `doc_name` supposed to be set dynamically somewhere?

Reply:
You're right. The `update-ci` target does not set this field in the CI configuration file. It was already done like that before. But sure, maybe we can do better by doing that automatically with this Makefile target.

Sorry again for this issue.

Reply:
Actually, looking now at the Makefile code, we are already doing this automatically when `DOCNAME` is set in the Makefile. But the problem is due to the spaces I have added between `doc_name` and the `:`. What a stupid mistake...sorry, I push a new PullRequest to fix this. I hope this time it will be OK.

Reply:
_See PullRequest https://github.com/ivoa-std/ivoatex/pull/143 for a fix_